content/webapp/components/OnThisPageAnchors/index.tsx

```diff
@@ -223,76 +381,144 @@ const OnThisPageAnchors: FunctionComponent<Props> = ({
     );
     buttonRef.current.setAttribute(
       'aria-expanded',
-      isListActive ? 'false' : 'true'
+      isListActive ? 'true' : 'false'
```

Comment:
`isListActive` itself would work I think?

Reply:
oh good point – would have to be stringy, but `${isListActive}` should do it I guess. Will update